release_notes.md

```diff
@@ -1,14 +1,9 @@
+
 ### Release notes
 <!-- Please add your release notes in the following format:
 - My change description (#PR)
 -->
-- Update Python Worker Version to [4.0.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.0.0)
```

Comment:
This is incorrect - my changes didn't go into 4.2.0 - Could you please validate?

Reply:
My changes went in 6 hours ago - https://github.com/Azure/azure-functions-host/pull/8156